docs/api/spec.md

```diff
@@ -1,9 +1,13 @@
 ---
 id: spec
-sidebar_label: API Specification
-title: API Specification (next)
+sidebar_label: Overview
+title: API Overview (next)
 ---
 
+The role of FDC3 API is to establish a baseline interface for interoperability between applications. Because FDC3 is largely an agreement between existing platforms and applications - standards should be optimized for ease of adoption rather than functional completeness. Functionality absent from a FDC3 specification is in no way a commentary on its importance.
```

Comment:
```suggestion
The role of FDC3 API is to establish a baseline interface for interoperability between applications. Because FDC3 is largely an agreement between existing platforms and applications, standards should be optimized for ease of adoption rather than functional completeness. Functionality absent from a FDC3 specification is in no way a commentary on its importance.
```